the-collab-lab/tcl-22-smart-shopping-list
# Added svg background to main element

## Description

Added svg background. Background has svg patterns of typical shopping items.

## Related Issue

#14 

## Acceptance Criteria

- Create a crisp clean background
- Make sure the background doesn't take user's attention away from the main aspect of the app

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |

### Before

![beforeBackground](https://user-images.githubusercontent.com/40097537/119594895-9eb9d180-bdaa-11eb-8ee6-39cac9c362c7.JPG)

### After

![afterBackground](https://user-images.githubusercontent.com/40097537/119594905-a5484900-bdaa-11eb-857e-bba873d6be1e.JPG)

## Testing Steps / QA Criteria

To test the feature, use one of the following ways:

1. Follow the [preview link](https://tcl-22-shopping-list--pr53-share-token-d4ye5443.web.app)
2. Launch the app locally from your terminal:
- Pull down this branch with `git pull origin kk-svgBackground` and check that branch out with `git checkout origin kk-svgBackground`
- Run npm start to launch the app in your browser at http://localhost:3000
